appservice/package.json

```diff
@@ -34,6 +34,7 @@
     "dependencies": {
         "@azure/abort-controller": "^1.0.4",
         "@azure/arm-appinsights": "^5.0.0-beta.4",
+        "@azure/arm-appservice-profile-2020-09-01-hybrid": "^2.0.0",
```

Comment:
Can this replace `@azure/arm-appservice` entirely?

Reply:
Especially until I can get to https://github.com/microsoft/vscode-docker/issues/3594

Reply:
We definitely can't replace the `@azure/arm-appservice` package with this one as the api-versions are different. I'm pretty sure that this only works with custom cloud. That being said, I understand the apprehension to adding more dependencies.

Reply:
What is it about the primary `@azure/arm-appservice` package that makes it incompatible with Azure Stack?

Reply:
https://github.com/microsoft/vscode-azurefunctions/issues/3242 The api-version itself which we can't dynamically change unfortunately.

Reply:
This seems like an easy thing for the SDK to handle...should we file an issue for it?

Reply:
Yeah or ideally they adopt the hybrid SDKs with the new REST sdks that are much smaller.